internal/vds/vds.cpp

```diff
@@ -83,6 +83,13 @@ bool unit_validation(axis_name ax, std::string const& zunit) {
     }
 };
 
+static response to_response(nlohmann::json const& metadata) {
```

Comment:
I was torn whether I should add it for data and decided against adding it as the function would add more lines that it would save. 

However, I agree that it would be more consistent to have this function for data as well. For data I would need a slightly different function signature as I would need to pass the pointer to the data and the size.

The naming indeed dropped the `metadata_` as this should be clear from the input. I have no strong preference on naming here. `json_axis` is somewhere in between in naming (to me) as it does not specify the "direction" of the conversion that explicitly. On the other hand, other `to_something` functions we have are members of classes such that the context is clearer than for the `to_response` functions so `to_response` may be too vague.